### PR TITLE
Inline banner on token list

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Shows a banner in the token list, linking to a guide on how to add custom tokens.
- Refactored code to move some items to the UI libs folder
- For testing purposes it's important to test for any Inline banner, Primary and Secondary button. Making sure it renders correctly.

Screenshot 2024-02-29 at 14 44 45

Screenshot 2024-02-29 at 14 44 54

### CODE REVIEW

File: libs/ui/src/pure/InlineBanner/index.tsx [hunk 1]
```diff
@@ -84,29 +86,48 @@ const Wrapper = styled.span<{
     gap: 10px;
     width: 100%;
 
-    ${({ theme }) => theme.mediaWidth.upToSmall`
+    /* ${({ theme }) => theme.mediaWidth.upToSmall`
       flex-flow: column wrap;
       gap: 16px;
-    `};
+    `}; */
```
Comment (alfetopito) [inline libs/ui/src/pure/InlineBanner/index.tsx:92]: Keep or remove?

File: libs/ui/src/pure/InlineBanner/index.tsx [hunk 2]
```diff
@@ -84,29 +86,48 @@ const Wrapper = styled.span<{
...
     line-height: 1.4;
     margin: auto;
     padding: 0;
     width: 100%;
     text-align: ${({ orientation = BannerOrientation.Vertical }) =>
       orientation === BannerOrientation.Horizontal ? 'left' : 'center'};
+
+    /* ${({ theme }) => theme.mediaWidth.upToSmall`
+      text-align: center;
+    `}; */
```
Comment (alfetopito) [inline libs/ui/src/pure/InlineBanner/index.tsx:127]: Same

File: libs/ui/src/pure/Icon/index.tsx [hunk 3]
```diff
@@ -2,7 +2,9 @@ import iconInformation from '@cowprotocol/assets/cow-swap/alert-circle.svg'
 import iconAlert from '@cowprotocol/assets/cow-swap/alert.svg'
 import iconDanger from '@cowprotocol/assets/cow-swap/alert.svg'
 import iconSuccess from '@cowprotocol/assets/cow-swap/check.svg'
-import { UI } from '@cowprotocol/ui'
+// import iconCoins from '@cowprotocol/assets/images/coins.svg'
```
Comment (alfetopito) [inline libs/ui/src/pure/Icon/index.tsx:5]: Same

**Review by alfetopito — APPROVED** [review]
Approval based on the code, did not test it

**Review by shoom3301 — APPROVED** [review]
Looks good!

**Comment by elena-zh** [comment]
Hey @fairlighteth , great job!

**Review by elena-zh — APPROVED** [review]

**Review by anxolin — APPROVED** [review]

**Pull request merged**